cluster/src/main/java/org/apache/iotdb/cluster/log/manage/PartitionedSnapshotLogManager.java

```diff
@@ -130,7 +130,7 @@ void collectTimeseriesSchemas(List<Integer> requiredSlots) {
       }
       Collection<TimeseriesSchema> schemas =
           slotTimeseries.computeIfAbsent(slot, s -> new HashSet<>());
-      IoTDB.schemaProcessor.collectTimeseriesSchema(sgPath, schemas);
+      //      IoTDB.schemaProcessor.collectTimeseriesSchema(sgPath, schemas);
```

Comment:
Remove this comment

Reply:
The whole ```Cluster``` module will be removed after release of new cluster.